lib/api/serialize.js

```diff
@@ -0,0 +1,32 @@
+// https://github.com/jquery/jquery/blob/2.1.3/src/manipulation/var/rcheckableType.js#L2
+// https://github.com/jquery/jquery/blob/2.1.3/src/serialize.js#L12-L13
+var _ = require('lodash'),
+    rCRLF = /\r?\n/g,
+    rcheckableType = /^(?:checkbox|radio)$/i,
+    rsubmitterTypes = /^(?:submit|button|image|reset|file)$/i;
+    // rsubmittable = /^(?:input|select|textarea|keygen)/i;
+exports.serializeArray = function() {
+  var Cheerio = this.constructor;
+  return this.find('input,select,textarea,keygen') // Taken from `rsubmittable`
+    .filter(function() {
+      // https://github.com/jquery/jquery/blob/2.1.3/src/serialize.js#L91-L94
+      // TODO: Come back to disabled
+      var $elem = Cheerio(this);
+      var type = $elem.attr('type');
+      return $elem.attr('name') && !$elem.is(':disabled') &&
```

Comment:
I wanted to add the `:matches` selector for some time now, which would allow the inclusion of this in the selector:

```css
:matches(input, select, textarea, keygen)
[name]
:not(:disabled)
:not(:checkbox, :radio)
:matches([checked], :not(

```